src/services/__tests__/anchor-service.test.ts

```diff
@@ -218,36 +217,6 @@ describe('anchor service', () => {
     expect(anchors[3].path).toEqual('1/1')
   })
 
-  test('Too few anchor requests', async () => {
```

Comment:
Originally I removed this test because `requestRepository.findAndMarkReady` determined if there aren't enough requests (and I tested it there), but since we are still in a mode where `anchorRequests` does both halves I have restored this test.